questionnaire-en.json

```diff
@@ -335,7 +326,7 @@
         },
         {
           "correctAnswer": "Yes",
-          "question": "Do you archive copies of the Compliance Artifacts of the Supplied Software?",
+          "question": "Do you have a documented procedure for archiving copies of Compliance Artifacts for the Supplied Software?",
           "number": "4.b",

```

Comment:
Somewhat of a substantial change - just pointing this out for others to review.

Reply:
This question is intended to address 4.1.2 of OpenChain Specification 2.0. The Specification explicitly asks for a documented procedure for archiving copies of the Compliance Artifacts of the Supplied Software. Therefore the previous use of "Do you archive copies of the Compliance Artifacts of the Supplied Software?" is correct but does not fully cover the intended scope of the specification requirement. While the marginally expands the scope of "Yes" or "No", I would contend that the change is not substantial enough to warrant any concern about previously conformant or in-process conformant entities, as the act of archiving copies if the Compliance Artifacts implicitly assumes and inherently requires a procedure to do so. But we should call it out explicitly moving forward.
https://wiki.linuxfoundation.org/_media/openchain/openchainspec-2.0.pdf